topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,316 @@
+> 2. **Bandage Image** {% icon tool %}:
+>    - *"Graphical Fragment Assembly"*: the Flye output file `Graphical Fragment Assembly` (not the "assembly_graph" file)
+>    - *"Node length labels"*: `Yes`
+>    - *Leave other settings as default*
+{: .hands_on}
+
+Your assembly graph may look like this:
+
+![assembly graph](../../images/sweet-potato-assembly-graph.png "Assembly graph of the nanopore reads, for the sweet potato chloroplast genome. In your image, some of the labels may be truncated; this is a known bug under investigation.")
+
+> ### {% icon question %} Question
+> What is your interpretation of this assembly graph?
+> > ### {% icon solution %} Solution
+> > One interpretation is that this represents the typical circular chloroplast structure: There is a long single-copy region (the node of around 78,000 bp), connected to the inverted repeat (a node of around 28,000 bp), connected to the short single-copy region (of around 11,000 bp). In the graph, each end loop is a single-copy region (either long or short) and the centre bar is the collapsed inverted repeat which should have about twice the sequencing depth.
+> {: .solution}
+{: .question}
+
+*Optional further steps:*
+* Repeat the Flye assembly with different parameters, and/or a filtered read set.
+* Try an alternative assembly tool, such as Canu or Unicycler.
+
+# Polish assembly
+
+Short illumina reads are more accurate the nanopore reads. We will use them to correct errors in the nanopore assembly.
+
+> ### {% icon hands_on %} Hands-on: Map reads
+> 1. **Map with BWA-MEM** {% icon tool %}:
+>    - *"Will you select a reference genome from your history"*: `Use a genome from history`
+>    - *"Use the following dataset as the reference sequence"*: `flye-assembly.fasta`
+>    - *"Algorithm for constructing the BWT index"*: `Auto. Let BWA decide`
+>    - *"Single or Paired-end reads"*: `Single`
+>    - *"Select fastq dataset"*: `sweet-potato-illumina-reduced.fastq`
+>    - *"Set read groups information?"*: `Do not set`
+>    - *"Select analysis mode"*: `Simple Illumina mode`
+{: .hands_on}
+
+* This maps the short reads to the assembly, and creates an alignment file.
+* Re-name this file `illumina.bam`
+
+> ### {% icon hands_on %} Hands-on: Polish
+> 1. **pilon** {% icon tool %}:
+>    - *"Source for reference genome used for BAM alignments"*: `Use a genome from history`
+>    - *"Select a reference genome"*: `flye-assembly.fasta`
+>    - *"Type automatically determined by pilon"*: `Yes`
+>    - *"Input BAM file"*: `illumina.bam`
+>    - *"Variant calling mode"*: `No`
+>    - *"Create changes file"*: `Yes`
+{: .hands_on}
+
+* This compares the short reads to the assembly, and creates a polished (corrected) assembly file.
+* There are two outputs: a `fasta` file and a `changes` file.
+* What is in the `changes` file?
+* Re-name the fasta output file `polished-assembly.fasta`
+* Find and run the tool called "Fasta statistics" on the original flye assembly and the polished version.
```

Comment:
This should probably be in the hands-on box too, right? (Judging that since it isn't one of the "optional further steps"

Reply:
thanks, have added this